Model/Validator.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace Amwal\Payments\Model;
+
+class Validator
+{
+    /**
+     * Validate BIN codes
+     *
+     * @param string $value
+     * @return bool
+     */
+    public function validateBinCodes(string $value): bool
+    {
+        $regex = '/^[0-9]{4,14}(?:,[0-9]{4,14})*$/';
```

Comment:
can you allow spaces before and after the commas as well?

Reply:
or even easier, keep the regex the same and just strip all spaces from $value before applying the regular expression
```suggestion
        $value = str_replace(' ', '', $value);
        $regex = '/^[0-9]{4,14}(?:,[0-9]{4,14})*$/';
```